TL/24_simple_design_patterns_singleton_page_object/Design_Patterns_README.md

```diff
@@ -0,0 +1,137 @@
+# Design Patterns
+***
+
+## Contents
+1. [Simple Design Patterns.](#simple-design-patterns)
+2. [Singleton](#singleton)
+3. [Page Object](#page-object)
+
+
+## 1. Simple Design Patterns.
```

Comment:
remove ~Simple~ or change to **About Design Patterns**

Reply:
done